pages/instructorQuestionManualGrading/instructorQuestionManualGradingNextInstanceQuestion.js

```diff
@@ -0,0 +1,30 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const {sqlDb, sqlLoader} = require('@prairielearn/prairielib');
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+router.get('/', (req, res, next) => {
+    const params = {
+        assessment_question_id: res.locals.assessment_question_id,
```

Comment:
Add a comment here that `assessment_question_id` is untrusted (has not been verified by an authz middleware) and so will need to be checked in the SQL. The `assessment_id` _has_ been verified by and authz middleware, so it can be trusted.